tests/MasterPeace/Bundle/BookBundle/Controller/BookControllerTest.php

```diff
@@ -13,12 +14,43 @@ class BookControllerTest extends WebTestCase
 
     public function testListAction()
     {
-        $client = static::createClient();
-        $this->logIn($client, [User::ROLE_ADMIN], 'KarolisM', 'password');
-
+        $client = $this->getLoggedClient();
         $crawler = $client->request('GET', '/book/');
         $this->assertCount(LoadBookData::getBookCount(), $crawler->filter('td'));
         $this->assertGreaterThan(0, $crawler->filter('td')->count());
         $this->assertGreaterThan(0, $crawler->filter('td:contains("Prisukamo paukščio kronikos")')->count());
     }
+
+    public function testCreateAction()
+    {
+        $client = $this->getLoggedClient();
+        $crawler = $client->request('GET', '/book/create');
+        $form = $crawler->filter('form[name=book]')->form();
+
+        $form['book[title]'] = 'Naujas Pavadinimas';
+        $form['book[author]'] = 'Autorius';
+        $form['book[year]'] = 2000;
+        $form['book[cover]'] = '';
+        $form['book[publisher]'] = 'Leidejas';
+        $form['book[isbnCode]'] = '9971-5-0210-0';
+
+        $client->submit($form);
+        $this->assertTrue($client->getResponse()->isRedirect());
+        $client->followRedirect();
+        $this->assertContains(
+            'Naujas Pavadinimas',
+            $client->getResponse()->getContent()
+        );
+    }
+
+    /**
+     * @return Client
+     */
+    private function getLoggedClient()
+    {
+        $client = static::createClient();
+        $this->logIn($client, [User::ROLE_ADMIN], '', '');
```

Comment:
Tai sakau logIn methode uzdek defaultinias reiksmes : logIn(Client $client, array $roles, $username = 'admin', $password = null) ir tada is cia trink tuos tuscius string, supratai ?:)